Lib/test/test_http_cookies.py

```diff
@@ -59,6 +59,39 @@ def test_basic(self):
             for k, v in sorted(case['dict'].items()):
                 self.assertEqual(C[k].value, v)
 
+    def test_obsolete_rfc850_date_format(self):
+        # Test cases with different days and dates in obsolete RFC 850 format
+        test_cases = [
```

Comment:
Can you include test vectors / examples from RFC 850 if any?

Reply:
This is really nitpicking, but I'd like you to also indicate something like:

```
[
 	# test vectors from RFC 850:
	...
	# some other test cases:
	...
]
```

This would help us detecting whether it's something from the RFC that is failing or not. If you can even include where you found it in the RFC, it would be easier for future debugging if any.